Tests/Unit/InMemoryRefreshTokenManagerTest.php

```diff
@@ -20,15 +20,17 @@ public function testClearExpired(): void
 
         timecop_freeze(new DateTime());
 
-        $testData = $this->buildClearExpiredTestData();
+        try {
+            $testData = $this->buildClearExpiredTestData();
```

Comment:
@HypeMC Does this line (and similar lines in other tests) need to be inside the `try` block? I don't see a case where building the data can throw an exception.

Reply:
@X-Coder264 Probably not, in theory, `new DateTime` could throw an exception if an invalid value is passed, but this is more for consistency I guess, everything between free & return are in a try..finally block.